src/archivist/fetcher/fullDomFetcher.js

```diff
@@ -1,9 +1,10 @@
-import puppeteer from 'puppeteer-extra';
+import puppeteer from 'puppeteer';
```

Comment:
Why did you import `puppeteer`?

From the doc of `puppeteer-extra`, it seems not necessary: 
```
// puppeteer-extra is a drop-in replacement for puppeteer,
// it augments the installed puppeteer with plugin functionality.
```

Reply:
puppeteer-extra does not export `errors` and especially the TimeoutError.
Could not find another way to do so

Reply:
ok